x-pack/test/alerting_api_integration/common/config.ts

```diff
@@ -185,6 +185,7 @@ export function createTestConfig(name: string, options: CreateTestConfigOptions)
           `--xpack.alerting.rules.run.actions.connectorTypeOverrides=${JSON.stringify([
             { id: 'test.capped', max: '1' },
           ])}`,
+          `--xpack.alerting.enableFrameworkAlerts=true`,
```

Comment:
Let's remember to ask folks to add this to their `kibana.dev.yml`files, after the merge, so we'll get some early testing of the initialization bits.